examples/cpp/counter-app/Makefile

```diff
@@ -8,7 +8,7 @@ ifndef $(abspath $(lastword $(MAKEFILE_LIST)))_MK
 $(abspath $(lastword $(MAKEFILE_LIST)))_MK := 1
 
 # uncomment to disable generate test related targets in this directory
-#TEST_TARGET := false
+# TEST_TARGET := false
```

Comment:
It seems I wasn't very clear in my previous comment. I don't believe any of the changes to this makefile are required if the `RECURSE_SUBMAKEFILES := false` change in `examples/cpp/Makefile#L23` works correctly based on the fix to bug #1574.